hpcgap/lib/mapping.gi

```diff
@@ -195,9 +202,15 @@ InstallGlobalFunction( Image, function ( arg )
 
         return [];
 
+      else
+        ErrorNoReturn( "couldn't match families of <elm> or <coll> and ",
+                       "Source(<map>),\n",
+                       "maybe <elm> or <coll> are not contained in ",
+                       "Source(<map>) or\naren't homogeneous lists," );
```

Comment:
Why is that `\n` in the middle of a line, not at the end? Perhaps format it like so:
```gap
        ErrorNoReturn( "couldn't match families of <elm> or <coll> and Source(<map>),\n",
                       "maybe <elm> or <coll> are not contained in Source(<map>) or\n",
                       "aren't homogeneous lists," );
```
Or if you are concerned about line width:
```gap
        ErrorNoReturn(
            "couldn't match families of <elm> or <coll> and Source(<map>),\n",
            "maybe <elm> or <coll> are not contained in Source(<map>) or\n",
            "aren't homogeneous lists," );
```

That said, I am not even sure it's such a good idea to insert manual line breaks... ???

Reply:
Yep, you're right. I'll drop the manual line breaks. It feels a bit weird.